_inc/lib/debugger/class-jetpack-cxn-tests.php

```diff
@@ -225,6 +225,12 @@ protected function test__wpcom_connection_test() {
 			return self::skipped_test( $name );
 		}
 
+		$id = Jetpack_Options::get_option( 'id' );
+
+		if ( ! is_int( $id ) ) {
```

Comment:
Do we have a legitimate reason why this could be happening? `is_int` will return `FALSE` if the returned value is not coerced to an integer, for example: `"42"`. I'm pretty sure we won't be storing the ID as a string anywhere, but I'm still worried that this might cause some false positives.

Reply:
That's fair. I don't know why it wouldn't be a number. @eliorivero any ideas since it was occurring on your site?

Reply:
I have definitely seen the ID stored as a string recently. I have also seen it stored as an integer recently.

:(

Reply:
Alrighty, I'm going to account for the 404 then and not worry about the type for now.

Reply:
48029775f